src/library_webgpu.js

```diff
@@ -1254,6 +1255,7 @@ var LibraryWebGPU = {
     }
 
     function makeColorState(csPtr) {
+      if (!csPtr) return undefined;
```

Comment:
the `csPtr` won't be null, it's just a pointer inside an array (see how it's invoked in `makeColorStates`). Instead we need to return `undefined` if `WGPUColorTargetState.format` is 0.